static/src/js/tree/Tree.jsx

```diff
@@ -30,6 +30,28 @@ class Tree extends React.Component {
       .every((ancestor) => this.state[ancestor]);
   }
 
+  renderTerm(code, term) {
+    const element = <Term term={term} />;
+
+    const inDefinition = this.props.definitionCodes.includes(code);
+
+    // conditionally wrap Term in an <href> element if there's a URL for it. We
+    // only want to link to codes included in the codelist so some codes will
+    // stay unlinked.
```

Comment:
> We only want to link to codes included in the codelist so some codes will stay unlinked.

Why do we want this?

Looks like this might be a result of building up `code_to_url` in the view.  I don't think we need to do this, since the URL can be determined from the coding system ID (which the JS doesn't yet have) and the concept code (which it does have).  As such, I think it's worth hard coding the URL structure in the JS.

Reply:
> Why do we want this?

I might have misunderstood something here, I'd assumed we only wanted the included codes linked.  Should I have linked all the codes?

Reply:
That matches the existing behaviour.

Reply:
Fixed in 67abb8a